.github/workflows/ci.yml

```diff
@@ -12,18 +12,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: ['lts/-1', 'lts/*', 'node']
```

Comment:
This is not a good idea. This make the CI dynamic, while we should really be controlling which Node.js versions we support.

Reply:
Fixed